content/cookbook/05-node/54-mcp-tools.mdx

```diff
@@ -17,11 +17,12 @@ let clientTwo;
 
 try {
   // Initialize an MCP client to connect to a `stdio` MCP server:
+  const transport = new StdioClientTransport({
+    command: 'node',
+    args: ['src/stdio/dist/server.js'],
+  });
```

Comment:
should we already change the bundling in this PR to have a single breaking change vs two?

Reply:
nevermind, saw that you did the split already. import might be missing